### ISSUE TITLE
Unsupported hide keyboard after tap on empty child listview space [Android]

### ISSUE THREAD

**Issue by hackerunet** [NativeScript/NativeScript#6376]
**Environment**
Provide version numbers for the following components (information can be retrieved by running `tns info` in your project folder or by inspecting the `package.json` of the project):
 - CLI: 4.2.4
 - Cross-platform modules:
 - Android Runtime: 4.1.3
 - iOS Runtime: 4.1.0
 - Plugin(s):

**Describe the bug**
To be able to hide the keyboard after tapping on any other space than a textField I added a tap action on the parent element of a ListView, this behavior works if the listView doesn't use itemtap action on the html. If I place another tap in the viewList with itemtap the itemtap behavior gets overwritten so it stops working. On iOS the problem is solved using iqKeyboardManager plugin but apparently on Android this is not supported. I also made this work adding a tap over another area other than the listView, so the issue here is that in Android NS is unable to determine the tap action inside the empty space of a listView which also has itemTap action declared and an item.

**To Reproduce**
Create a ListView with an item and add a basic itemtap, add a tap action over a parent element and try in the tap action to dismissSoftKeyboard on android.

**Expected behavior**
After tap on the empty space of a listView the keyboard should hide automatically calling the dismissSoftKeyboard action

**Sample project**
https://play.nativescript.org/?template=play-ng&id=F805hD

**Additional context**
We are not sure why ListView only supports itemtap or tap on its child elements, maybe if there is a way to hide the keyboard globally in the app for android after tapping outside a textField would be great, so far I haven't found a solution for this in the case of hiding the keyboard after tapping outside a listView or inside the ListView empty space

**Comment by tsonevn** [comment]
Hi @hackerunet,
I tested the attached sample project and found that in your case you can set up itamTap and tap events on the ListView component. From the test, it seems that both events will be fired when you've tapped on an item, however,  if you touch the empty ListView space, only the tap will be fired. Check out the project [here](https://play.nativescript.org/?template=play-ng&id=F805hD&v=2). You can check when just the tap is executed and to call [dismissSoftInput](https://docs.nativescript.org/core-concepts/utils) method provided by the **utils/utils** module, which will close the keyboard.

**Comment by JoseJoya** [comment]
I'm Testing this project on IOS and the itemTap Event does not work as expected!, only tap event was executed

**Comment by hackerunet** [comment]
Yes confirmed, on Android the issue was "t" instead of "T' on itemTap function, but on iOS is not working. this was tested on iOS 10+ @JoseJoya @tsonevn

**Comment by hackerunet** [comment]
Any news why itemTap doesn't work with tap on ListViews for iOS ?

**Comment by tsonevn** [comment]
Hi @hackerunet,
I have researched further the case, however, was unable to find a way to set up both itamTap and tap on the ListView for iOS. I will mark this as a bug, and we will investigate further how this can be achieved. I can not commit to the exact time when this will be done. For now, I would suggest keeping track on the issue for further info.

**Comment by hackerunet** [comment]
I appreciate all your effort, will be looking into this and expecting to have a solution soon. thanks.

**Comment by romandragan** [comment]
@tsonevn @hackerunet, any progress on that?

**Comment by hackerunet** [comment]
No, abandoned nativescript and replaced with Flutter. Thanks.
